test/WebJobs.Script.Tests/TestScripts/CSharp/NotificationHubOut/function.json

```diff
@@ -0,0 +1,13 @@
+﻿{
+  "bindings": [
+    {
+      "type": "manualTrigger",
+      "direction": "in"
+    },
+    {
+      "type": "notificationhub",
+      "name": "messageProperties",
```

Comment:
same here. messageProperties is key value pairs that will be used to build templateNotification